Lib/multiprocessing/reduction.py

```diff
@@ -68,12 +68,19 @@ def dump(obj, file, protocol=None):
     __all__ += ['DupHandle', 'duplicate', 'steal_handle']
     import _winapi
 
-    def duplicate(handle, target_process=None, inheritable=False):
+    def duplicate(handle, target_process=None, inheritable=False,
+                  *, source_pid=None):
         '''Duplicate a handle.  (target_process is a handle not a pid!)'''
```

Comment:
A bit annoying. One argument is a process handle and the other is a pid?

Reply:
steal_handle() also has a weird API, but I agree that it's not a good reason to continue that :-) I changed the parameter to source_process, and moved the pid => handle dance to the caller.